src/components/buttons/ha-call-api-button.js

```diff
@@ -1,70 +1,67 @@
-import { html } from '@polymer/polymer/lib/utils/html-tag.js';
-import { PolymerElement } from '@polymer/polymer/polymer-element.js';
-
+import { LitElement, html } from '@polymer/lit-element';
 
 import './ha-progress-button.js';
-import EventsMixin from '../../mixins/events-mixin.js';
+import fireEvent from '../../common/dom/fire_event.js';
 
-/*
- * @appliesMixin EventsMixin
- */
-class HaCallApiButton extends EventsMixin(PolymerElement) {
-  static get template() {
+class HaCallApiButton extends LitElement {
+  render() {
     return html`
-    <ha-progress-button id="progress" progress="[[progress]]" on-click="buttonTapped" disabled="[[disabled]]"><slot></slot></ha-progress-button>
-`;
+      <ha-progress-button
+        .progress="${this.progress}"
+        @click="${this._buttonTapped}"
```

Comment:
Listen to events on this component (in JS: `el.addEventListener('click', () => this.buttonTapped())`)